src/main/java/teammates/ui/webapi/action/ActionFactory.java

```diff
@@ -106,8 +106,8 @@
         map(ResourceURIs.COURSE_EDIT_DETAILS, GET, GetCourseEditDetailsAction.class);
         map(ResourceURIs.COURSE_EDIT_DETAILS_SAVE, PUT, SaveCourseEditDetailsAction.class);
         map(ResourceURIs.COURSE_DELETE, DELETE, DeleteCourseAction.class);
-        map(ResourceURIs.COURSE_EDIT_INSTRUCTOR_DETAILS, POST, EditInstructorInCourseAction.class);
-        map(ResourceURIs.COURSE_ADD_INSTRUCTOR, PUT, CreateInstructorInCourseAction.class);
+        map(ResourceURIs.INSTRUCTORS, PUT, UpdateInstructorAction.class);
+        map(ResourceURIs.INSTRUCTORS, POST, CreateInstructorAction.class);
```

Comment:
`INSTRUCTORS` or `INSTRUCTOR`?

Reply:
Yeah your right! Sry about that... fixed!